online-inference/stable-diffusion/00-optional-s3-secret.yaml

```diff
@@ -1,22 +1,22 @@
 apiVersion: v1
 data:
-  access_key: Q0hBTkdFTUU=
+  access_key: UkVQTEFDRU1F
 kind: Secret
 metadata:
   name: s3-access-key
 type: Opaque
 ---
 apiVersion: v1
 data:
-  secret_key: Q0hBTkdFTUU=
+  secret_key: UkVQTEFDRU1F
 kind: Secret
 metadata:
   name: s3-secret-key
 type: Opaque
 ---
 apiVersion: v1
 data:
-  url: Q0hBTkdFTUU=
+  url: UkVQTEFDRU1F
```

Comment:
Can these values be <CW_OBJECT_STORAGE_ACCESS_KEY_BASE64>... instead of UkVQTEFDRU1F